src/main/kotlin/mods/eln/sixnode/ElectricalVuMeter.kt

```diff
@@ -32,7 +32,7 @@ import org.lwjgl.util.Color
 import java.io.DataInputStream
 import java.io.DataOutputStream
 import java.io.IOException
-import java.util.HashMap
+import java.util.*
```

Comment:
This is probably the fault of a linter change; can we make sure that we have fully qualified imports?

Reply:
... I didn't even tell IntelliJ to do any auto corrections in those files, fixed the warnings manually, how on Earth???  Might want to change the formatter options for the project?